libcontainer/cgroups/utils_test.go

```diff
@@ -661,3 +664,42 @@ func TestConvertBlkIOToIOWeightValue(t *testing.T) {
 		}
 	}
 }
+
+// TestRemovePathEROFS is to test remove a non-exist dir in a ro mount point.
+// The similar issue example: https://github.com/opencontainers/runc/issues/4518
+func TestRemovePathEROFS(t *testing.T) {
+	dirFrom, err := os.MkdirTemp(".", "from-")
+	if err != nil {
+		t.Skip("can't create a tmp dir")
+	}
+	defer func() {
+		_ = unix.Rmdir(dirFrom)
+	}()
+	dirTo, err := os.MkdirTemp(".", "to-")
+	if err != nil {
+		t.Skip("can't create a tmp dir")
+	}
+	defer func() {
+		_ = unix.Rmdir(dirTo)
+	}()
```

Comment:
I _think_ this can be replaced with
```suggestion
	dirFrom := t.TempDir()
	dirTo := t.TempDir()
```

Reply:
or even
```go
dirTo := t.TempDir()
err := unix.Mount(t.TempDir(), dirTo, "", unix.MS_BIND, "")
```